### DISCUSSION TITLE
Lagoon in DOKS

### DISCUSSION THREAD

**Discussion by icasimpan** [amazeeio/lagoon#1995] [General] [answered]
Hi,

I'm interested in using lagoon using Kubernetes in DigitalOcean's managed kubernetes (aka DOKS).

I understand that official support is still in v2.0 but I'm wondering if the current code as of v1.7.1 already works though with some kinks?

Appreciate the hints.

Thank you.

**Comment by icasimpan** [comment]
Been searching the issue queue and found https://github.com/amazeeio/lagoon/issues/100 as a good start.

**Comment by tobybellwood** [comment, marked as answer]
Hi @icasimpan - thanks for reaching out.  We're not quite ready yet for running Lagoon itself on Kubernetes - that's our Lagoon 2.0 release currently being worked on - we'll commence converting our services and images to helm charts shortly.

What we are able to do (as of 1.4.0) is deploy from an existing Lagoon into a Kubernetes cluster.  We have currently got customers on AKS, EKS and shortly GKE, all of which are managed by us.  Local Lagoon development also does this - it hosts Lagoon in a Docker Compose stack, and then uses that to deploy to a k3s install running on your local.

We've looked briefly at DOKS in the past - and think we can manage to deploy to it (the main difference to our other setups is the lack of File storage - it has block and object).

If you want to talk more to us about it, where we're at, and what's possible - we're always available at https://amazeeio.rocket.chat/

**Comment by icasimpan** [comment]
Thank you @tobybellwood. I guess the best option then is to wait for Lagoon 2.0.

**Reply by tobybellwood** [reply]
To run it yourself yes, but we'd be more than happy to talk now about how we could run our Lagoon to deploy your sites into your DOKS.
